molecule_vagrant/modules/vagrant.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/python
+#!/usr/bin/python3
```

Comment:
What about putting python3 changes in a different commit ?

Reply:
Fixed in 72aae24.